Openvstorage/Framework documentation/django_stack.md

```diff
@@ -0,0 +1,49 @@
+
+
+## Django stack
+As could be expected, our django stack is located in the `webapps.api` folder. We only use a subset of the whole Django functionality.
```

Comment:
`As could be expected` can be left.
We use Django for:
- Authentication (although the backend is customized)
- Django REST framework views (enhanced with decorators)
- Serialization (customized)

Reply:
Also explain that the choice of Django isn't the best as Django relies heavily on its own ORM while the OVS Framework has it's own. Therefore we work around most Django features by injecting our own truths into it. A nasty thing to do.
The better choice would be Flask because it does not rely on any ORM whatsoever. It's also the way forward if we'd update the python version as the Django REST Framework future versions are harder to customize.